packages/interactive_media_ads/pigeons/interactive_media_ads_android.dart

```diff
@@ -0,0 +1,717 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// // ignore_for_file: avoid_unused_constructor_parameters
```

Comment:
Why is everything in this file commented out?

Reply:
This uses features from https://github.com/flutter/packages/pull/6371, so it would break tests without it commented out. But I'll go ahead and uncomment it for now so it can be reviewed while I still wait for https://github.com/flutter/packages/pull/6371 to land.